src/view/frontend/web/js/billing-address-autocomplete.js

```diff
@@ -0,0 +1,251 @@
+
+            if (self.moduleEnabled !== '1' && self.googleMapError) {
+                return;
+            }
+
+            setTimeout(function () {
+                var domID = uiRegistry.get(self.formComponent + '.street').elems()[0].uid;
+                var street = $('#' + domID);
+
+                //SHQ18-260
+                var observer = new MutationObserver(function () {
+                    observer.disconnect();
+                    $("#" + domID).attr("autocomplete", "new-password");
+                });
+
+                street.each(function () {
+                    var element = this;
+
+                    observer.observe(element, {
+                        attributes: true,
+                        attributeFilter: ['autocomplete']
+                    });
+
+                    self.autocomplete = new google.maps.places.Autocomplete(
+                        /** @type {!HTMLInputElement} */(this),
+                        {types: ['geocode']}
+                    );
+                    self.autocomplete.addListener('place_changed', self.fillInAddress.bind(self));
+
+                });
+                $('#' + domID).focus(self.geolocate.bind(this));
+
+                self.isAutocompleteInitialized(true);
+            }, 1000);
+        },
+
+        fillInAddress: function () {
+            var place = this.autocomplete.getPlace();
+
+            var street = [];
+            var region  = '';
+            var streetNumber = '';
+            var city = '';
+            var postcode = '';
+            var postcodeSuffix = '';
+
+            for (var i = 0; i < place.address_components.length; i++) {
+                var addressType = place.address_components[i].types[0];
+                if (this.componentForm[addressType]) {
+                    var value = place.address_components[i][this.componentForm[addressType]];
+                    if (addressType == 'subpremise') {
+                        streetNumber = value + '/';
+                    } else if (addressType == 'street_number') {
+                        streetNumber = streetNumber + value;
+                    } else if (addressType == 'route') {
+                        street[1] = value;
+                    } else if (addressType == 'administrative_area_level_1') {
+                        region = value;
+                    } else if (addressType == 'sublocality_level_1') {
+                        city = value;
+                    } else if (addressType == 'postal_town') {
+                        city = value;
+                    } else if (addressType == 'locality' && city == '') {
+                        //ignore if we are using one of other city values already
+                        city = value;
+                    } else if (addressType == 'postal_code') {
+                        postcode = value;
+                        var thisDomID = uiRegistry.get(this.formComponent + '.postcode').uids
```

Comment:
Typo, the correct code should: 

`var thisDomID = uiRegistry.get(this.formComponent + '.postcode').uid`

Reply:
If the typo has fixed, then the PR is working great. Thanks, Jerry!